app/android/.gitignore

```diff
@@ -0,0 +1,11 @@
+gradle-wrapper.jar
+/.gradle
+/captures/
+/gradlew
+/gradlew.bat
+/local.properties
+GeneratedPluginRegistrant.java
+
+# Remember to never publicly share your keystore.
+# See https://flutter.dev/docs/deployment/android#reference-the-keystore-from-the-app
```

Comment:
This is similar to what we'll have to do with VPN, useful to save this link for future reference somewhere else

Reply:
Added this link to discord under useful links